pkg/operation/botanist/health_check.go

```diff
@@ -615,6 +617,7 @@ func (b *Botanist) checkSystemComponents(
 	shootDaemonSetLister kutil.DaemonSetLister,
 	extensionConditions []extensionCondition,
 ) (*gardencorev1beta1.Condition, error) {
+	ctx := context.TODO()
```

Comment:
Can we add the context as the first parameter instead of creating one in here?

Reply:
Done, although it doesn't help too much. The whole file is not used `context` yet, and there must be dedicated time spent to implement it cleanly (#1648)

Reply:
Ok, got it. Thanks, anyways.